src/scribe_data/cli/cli_utils.py

```diff
@@ -27,8 +27,6 @@
 
 from scribe_data.utils import DEFAULT_JSON_EXPORT_DIR
 
-# MARK: CLI Variables
```

Comment:
Please don't remove marks from the code in the future @Collins-Webdev as these are meant to make the files more manageable to navigate.